src/components/Search.js

```diff
@@ -49,23 +58,42 @@ export default class Search extends Component {
             Explore!
           </Button>
         </Form>
-        <Card style={{ width: "18rem" }} className='card'>
+        <Card style={{ width: "70rem" }} className='card'>
           <Card.Img
             variant='top'
-            src={`https://maps.locationiq.com/v3/staticmap?key=${process.env.REACT_APP_LOCATIONIQ_KEY}&center=${this.state.locationObject.lat},${this.state.locationObject.lon}&zoom=18`}
+            src={`https://maps.locationiq.com/v3/staticmap?key=${process.env.REACT_APP_LOCATIONIQ_KEY}&center=${this.state.locationObject.lat},${this.state.locationObject.lon}&zoom=16&size=1280x720`}
           />
           <Card.Body>
             {this.state.locationObject.display_name ? (
               <p>
                 <Card.Title>
                   {this.state.locationObject.display_name}
                 </Card.Title>
-                ,{this.state.locationObject.lat},{this.state.locationObject.lon}{" "}
+                <ListGroup className='list-group-flush' className='ListGroup'>
+                  <ListGroupItem className='ListGroup'>
+                    Latitude {this.state.locationObject.lat}
+                  </ListGroupItem>{" "}
+                  <ListGroupItem className='ListGroup'>
+                    {" "}
+                    Longitude
+                    {this.state.locationObject.lon}
+                  </ListGroupItem>
+                </ListGroup>
               </p>
             ) : (
               <p>Search for a city to explore</p>
             )}
-            {this.state.error && <p>There was an error with your request</p>}
+            {this.state.error && (
+              <Alert
+                varient='danger'
+                className='error'
+                onClose={() => this.setState({ error: false })}
```

Comment:
I couldn't get onClose to work on mine. This is useful to see a working example.